src/gui/DatabaseWidget.cpp

```diff
@@ -1247,7 +1247,7 @@ void DatabaseWidget::reloadDatabaseFile()
             if (m_databaseModified) {
                 // Ask if we want to merge changes into new database
                 QMessageBox::StandardButton mb = MessageBox::question(this, tr("Merge Request"),
-                                     tr("The database file has changed and you have unsaved changes."
+                                     tr("The database file has changed and you have unsaved changes.\n"
```

Comment:
Please use append(). Newlines don't need to be translated and including it in the translation source string breaks existing translations.